lib/services/northBound/deviceGroupAdministrationServer.js

```diff
@@ -267,21 +274,32 @@ function loadContextRoutes(router) {
     router.post(
         '/iot/services',
         restUtils.checkRequestAttributes('headers', mandatoryHeaders),
+        restUtils.checkBody(templateGroupService),
+        handleCreateDeviceGroup
+    );
+
+    router.post(
+        '/iot/groups',
+        restUtils.checkRequestAttributes('headers', mandatoryHeaders),
         restUtils.checkBody(templateGroup),
         handleCreateDeviceGroup
     );
```

Comment:
Needed 2 `router.post()` entries in order to allow to use a different template for each query

Reply:
I got that. Maybe CheckBody should take care of it. If it can do comlex checkings.

Reply:
"revalidator" dependency doesn't allow conditional checking. The point here is that limits on  auxiliar libs shouldn't drive to special "code structures" at the top level code. Top Level code is more important than a limited validation library. Modify checkBody Function to accept arrays of options (os maybe just two options), make a reduction function... I think uniformity at this level is more important than check two options of JSON.